src/big_mood_detector/infrastructure/parsers/json/fitbit_parsers.py

```
@@ -0,0 +1,59 @@
+            "sleep_records": self.sleep_parser.parse(data),
+            "activity_records": self.activity_parser.parse(data),
+            "heart_rate_records": self.heart_rate_parser.parse(data),
+        }
+
+    def parse_file(self, file_path: str | Path) -> dict[str, list[Any]]:
+        """Parse Fitbit data from a JSON file path."""
+        try:
+            with open(file_path, encoding="utf-8") as file_handle:
+                payload = json.load(file_handle)
+        except (OSError, UnicodeDecodeError, json.JSONDecodeError) as error:
+            raise ValueError(f"Failed to parse Fitbit JSON file: {file_path}") from error
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)